bin/check-ssl-qualys.rb

```diff
@@ -107,11 +129,37 @@ def ssl_check(from_cache)
 
   def ssl_recheck
     1.upto(config[:num_checks]) do |step|
-      json = ssl_check(step != 1)
+      p "step: #{step}" if config[:debug]
```

Comment:
If these debug statements are intended to be long term, wouldn't `puts` be better?

Reply:
well `p` shows the objects by doing an inspect while `puts` does a `.to_s` which makes it harder to spot issues like improper types. For debug purposes it is much better for normal user interaction `puts` is generally preferable.